app/controllers/repos_controller.rb

```diff
@@ -95,17 +113,21 @@ def show_releases
         columns = ChartBuilder.new(benchmark_runs).build_columns do |benchmark_run|
           environment = YAML.load(benchmark_run.environment)
 
+          # generate the ruby_version object
+          config = {version: benchmark_run.initiator.version}
```

Comment:
`{ version: benchmark_run.initiator.version }` according to the ruby style guide